README.md

```diff
@@ -258,12 +258,12 @@ await service.removeCoupon('COUPON');
 await service.removeGiftCertificate('GIFT');
 ```
 
-### Initialize spam protection
+### Execute spam protection check
 
 You can also enable bot protection to prevent bots and other types of automated abuse from creating orders. Note that enabling this feature increases checkout friction, which may affect conversions. As such, we recommend leaving this feature out if your store is not encountering bots.
 
 ```js
-await service.initializeSpamProtection({ containerId: 'spamProtectionContainer' });
+await service.executeSpamCheck();
```

Comment:
might be worth explaining when this is supposed to be called (https://github.com/bigcommerce/checkout-sdk-js/issues/856)

Reply:
@capsula4 Does this method automatically handle refreshing the recaptcha? If the shopper is on the payment step for 20 mins, will the security check expire when they try to purchase, or is the sdk handling automatically refreshing the token?